Mathlib.lean

```diff
@@ -4596,7 +4596,7 @@ import Mathlib.Topology.PartialHomeomorph
 import Mathlib.Topology.PartitionOfUnity
 import Mathlib.Topology.Perfect
 import Mathlib.Topology.QuasiSeparated
-import Mathlib.Topology.RestrictGenTopology
+import Mathlib.Topology.RestrictGen
```

Comment:
Why do you move a file in the same PR?

Reply:
Oh I guess that should be a preliminary PR